libcxx/utils/libcxx/header_information.py

```diff
@@ -161,6 +161,10 @@ def is_header(file):
     ]
 
 
+def is_public_header(header):
+    return "__" not in header and not header.startswith("ext/")
```

Comment:
`not header.startswith("ext/")` is to prevent appearing `ext/*` headers in 
https://github.com/llvm/llvm-project/blob/main/libcxx/include/__std_clang_module

But I'm not sure that is intentional or not.